features/foutafhandeling.feature

```diff
@@ -5,141 +5,65 @@ Functionaliteit: Afhandeling van fouten
-        | title  | Ten minste één parameter moet worden opgegeven. |
-        | status | 400                                             |
-        | code   | paramsRequired                                  |
-        | detail | <error detail>                                  |
-        En bevat de response geen invalidParams
+        | name        | code   | reason                |
+        | <parameter> | <code> | <reason omschrijving> |
 
         Voorbeelden:
-        | path                   | error detail                                                                                                                                          |
-        | /adressen              | Precies 1 parameter van pandIdentificatie, adresseerbaarObjectIdentificatie, zoekresultaatIdentificatie, postcode of huisnummer moet worden opgegeven |
-        | /adresseerbareobjecten | Precies 1 parameter van nummeraanduidingIdentificatie, pandIdentificatie, pandIdentificaties of bbox moet worden opgegeven                            |
-        | /panden                | Precies 1 parameter van adresseerbaarObjectIdentificatie, nummeraanduidingIdentificatie, locatie of bbox moet worden opgegeven                        |
+        | path      | query string                                        | detail                                                                                           | parameter | code             | reason omschrijving                   |
+        | /panden   | ?locatie=abc,def                                    | Ongeldige waarde [abc,def] opgegeven voor parameter [locatie]                                    | locatie   | number           | Waarde is geen geldig decimaal getal. |
+        | /panden   | ?locatie=194189.983                                 | Bad request.                                                                                     | locatie   | minItems         | Array bevat minder dan 2 items.       |
+        | /panden   | ?locatie=194189.983,abc                             | Ongeldige waarde [194189.98399146,abc] opgegeven voor parameter [locatie]                        | locatie   | number           | Waarde is geen geldig decimaal getal. |
+        | /panden   | ?locatie=5.96072575,52.18450437                     | Ongeldige waarde [194189.98399146,465870.76799701] opgegeven voor parameter [locatie]            | locatie   | geometryMismatch | Waarde is niet conform opgegeven CRS. |
+        | /panden   | ?bbox=a,b,c,d                                       | Ongeldige waarde [a,b,c,d] opgegeven voor parameter [bbox]                                       | bbox      | number           | Waarde is geen geldig decimaal getal. |
+        | /panden   | ?bbox=134647.123,457842.456                         | Bad request.                                                                                     | bbox      | minItems         | Array bevat minder dan 4 items.       |
```

Comment:
@strijm waarom wordt bij sommige fouten in bbox of locatie als detail een redelijk nietszeggend "Bad request." gegeven en in andere de waarde en fout genoemd?

Reply:
Omdat die fout al bestaat in de HC common en die gebruiken we ook voor locatie en bbox parameter.
Zie ook opmerkingen hierboven.